webpack/assets/javascripts/react_app/common/Tour/TourActions.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable promise/prefer-await-to-then */
+import API from '../../API';
+import {
+  TOUR_UPDATE_STATUS,
+  TOUR_GET_STATUSES,
+  TOUR_START_RUNNIG,
+  TOUR_STOP_RUNNING,
+  TOUR_REGISTER,
+  TOURS_REQUESTS,
+} from './TourConstants';
+
+export const updateAsSeen = id => async dispatch =>
+  API.post('/user_preferences', {
```

Comment:
This request will fail if the user is not logged in (I just happened to pick `LoginPage` as the page to test out the tour. :laughing:)  Not sure it's worth bothering, but may think about skipping this part if the user isn't authenticated?